doc/REQUIRE

```diff
@@ -0,0 +1,3 @@
+Compat 0.9.5 0.9.5+
+DocStringExtensions 0.3.0 0.3.0+
+Documenter 0.8.2 0.8.2+
```

Comment:
@tkelman, I assume you did mean pinning *all* the dependencies rather than just Documenter?